__tests__/index.test.js

```diff
@@ -108,6 +108,38 @@ describe('/api/articles', () => {
         });
       });
   });
+  test('GET:200 Responds with specific articles (articles array) of article objects when topic is specified', () => {
+    const topic = "mitch"
+    return request(app)
+      .get('/api/articles')
+      .query({topic})
+      .expect(200)
+      .then((response) => {
+          expect(response.body.articles.length).toBe(12);
+        response.body.articles.forEach((article) => {
+          expect(article).toMatchObject({
+            article_id: expect.any(Number),
+            title: expect.any(String),
+            topic: expect.any(String),
+            author: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            article_img_url : expect.any(String),
+            body: expect.any(String)
+          })
+        });
+      });
+  });
```

Comment:
Another test that should be added here is a test to ensure when given a topic that does exist but has no associated articles we still receive an empty array.

Reply:
Added this test.  This change will be part of re-commit of Task 11.